### DISCUSSION TITLE
Ender 3 S1 Pro Temperature only 260?

### DISCUSSION THREAD

**Discussion by gitfve** [mriscoc/Ender3V2S1#1139] [General]
Ender 3 S1 Pro.
Any logic that the fw cant set higher then 260dC
Spec for the printer is: Max Nozzle Temperature, 300 ºC

**Comment by mriscoc** [comment]
For Sprite Extruder Pro use the T13 version.

**Comment by gitfve** [comment]
Now I made a new compiling with BLT, UBL, T13(high temp) for my Ender3S1**Pro** with the F4 chip, only 256KB Rom.

The rom is tested in the printer with some print jobs, some have printed 90 minutes other 10 minutes, and all work nice, the run_out sensor too.

**Thanks Miguel(mriscoc) for all the work you have put in this, and the help in the forum.**

I used all the latest file from mriscoc/Ender3V2S1/releases and mriscoc/Special_Configurations

**This is my config:**
You can use the S1.py as a templates and strip the other things out.
CreateConfigs.Generate('E3S1Pro-F4-T13-UBL-b20230904', ['Ender3S1','301F4','BLT','UBL','T13','F4SaveRom'])

The file F4SaveRom.json must be placed in the _features dir.

**This is my F4SaveRom.json:**
```
{
"Configuration.h" : [
  {
    "op": "Disable",
    "searchfor": "BAUD_RATE_GCODE"
  },
  {
    "op": "Disable",
    "searchfor": "INDIVIDUAL_AXIS_HOMING_SUBMENU"
  },
  {
    "op": "Disable",
    "searchfor": "HAS_CUSTOM_COLORS"
  },
  {
    "op": "Disable",
    "searchfor": "PLR_TUNE_ITEM"
  }
],
"Configuration_adv.h" : [
  {
    "op": "Disable",
    "searchfor": "LCD_BACKLIGHT_TIMEOUT_MINS"
  },
  {
    "op": "Disable",
    "searchfor": "STATUS_MESSAGE_TIMEOUT_SEC"
  },
  {
    "op": "Enable",
    "searchfor": "SDCARD_READONLY"
  },
  {
    "op": "Disable",
    "searchfor": "POWER_LOSS_RECOVERY"
  },
  {
    "op": "Disable",
    "searchfor": "LONG_FILENAME_HOST_SUPPORT"
  },
  {
    "op": "Disable",
    "searchfor": "LONG_FILENAME_WRITE_SUPPORT"
  },
  {
    "op": "Disable",
    "searchfor": "CANCEL_OBJECTS"
  }  
],
"Version.h" : [
]   
}
```
